src/ApiStatus.ts

```diff
@@ -50,19 +49,19 @@ export default class ApiStatus {
         if (!this.isApiStatusCommand(content))
             return;
 
-        let apiStatus = await this.getApiStatus();
+        const apiStatus = await this.getApiStatus();
 
-        let fields: Array<{name: string, value: string, inline: boolean}> = [];
-        for (let api in apiStatus.api){ 
-            fields.push({name: api, value: apiStatus.api[api], inline: true})
+        const fields: Array<{name: string, value: string, inline: boolean}> = [];
```

Comment:
Minor nitpick, but this should technically also be `{ name: string, value: string, inline: boolean }[]`. You should be able to omit the type entirely though, the latest TypeScript versions infer array types based on what kind of elements you put in them (although it might not infer this properly since it isn't guaranteed to be filled).

Reply:
It infers as `any[]`. I'll change it to `{name: string, ...}[]`